README.md

```diff
@@ -44,6 +44,32 @@ Please help enhance the extension by suggesting new default characters, through
 
 You can find all characters in [Unicode Table](https://unicode-table.com/en/).
 
+## Language-specific gremlins characters
+
+You can override the characters for a specific language by configuring them in the `gremlins.characters` property of the language-specific settings key (e.g. `[markdown]` for Markdown files).
+
+> More information about language specific settings can be found in the [Language specific editor settings](https://code.visualstudio.com/docs/getstarted/settings#_language-specific-editor-settings) VSCode documentation page.
+
+As an example, the following snippet adds the "U+000C" (form feed) character and disables the "U+00A0" (non-breaking space) character for markdown files:
+
+```jsonc
+"[markdown]": {
+  "gremlins.characters": {
+    // Add the form feed character for markdown files
+    "000c" : {
+      "zeroWidth": true,
+      "description": "FORM FEED (FF)",
+      "backgroundColor": "rgba(255,127,80,.5)",
+      "overviewRulerColor": "rgba(255,127,80,1)",
```

Comment:
Could we use the `level` property here, or did you use color properties to make it different from the available levels?

Reply:
I have no idea how `backgroundColor` ended up in the example; it's not overridable in the code and _does_ come from `level`.  I'll remove it from the examples.  On the other hand, `overviewRulerColor` has been in the code a long time.

Reply:
> I have no idea how `backgroundColor` ended up in the example; it's not overridable in the code and does come from level. I'll remove it from the examples. 

👍

> On the other hand, `overviewRulerColor` has been in the code a long time.

I think I was not clear enough, sorry.

Being able to define `backgroundColor` and `overviewRulerColor` in settings is fine.

I just suggested using `level` instead for this language specific configuration.

Reply:
Ah, I follow you.  I've updated the example to just use `level`.

Reply:
Perfect!